tests/list.cpp

```diff
@@ -425,6 +425,8 @@ TEST_CASE("list") {
             REQUIRE_MOVES(change, {1, 2});
         }
 
+#if 0
+        // FIXME irrelevant?
```

Comment:
Yes, everything related to moving rows in tables should just be deleted.

Reply:
Deleted